app/lib/backend/storage.rb

```diff
@@ -6,7 +6,7 @@ class Storage < ::System::RedisPool
     def self.parse_config
       config = File.read("#{Rails.root}/config/backend_redis.yml")
       config = ERB.new(config).result(binding)
-      config = YAML.load(config)
+      config = YAML.safe_load(config, aliases: true)
```

Comment:
why call `YAML.safe_load(config, aliases: true)` instead of `YAML.load(config, aliases: true)`

I understand from your links that safe mode is the default now.

Reply:
Yeah, true, I think in Psych 4.x we can call `load` safely.